decoder/impeghd_obj_ren_dec.c

```diff
@@ -119,28 +119,20 @@ static IA_ERRORCODE impeghd_flex_spk_2_ls_geometry(ia_cicp_ls_geo_str *array_cic
   WORD32 i;
   WORD32 num_speakers;
   WORD32 ch_idx;
-  WORD32 cicp_idx = ptr_ref_spk_layout->cicp_spk_layout_idx;
   ia_flex_spk_data_str *ptr_flex_spk = &ptr_ref_spk_layout->str_flex_spk;
-  if (cicp_idx == 0 || cicp_idx == 8 || cicp_idx > NUM_LS_CFGS)
```

Comment:
Any specific reason why this error check is removed?

Reply:
removed the check as, cicp_idx isn't being used anywhere within the function.